src/WeekCalendar/CalendarCells.tsx

```diff
@@ -18,12 +26,15 @@ const CalendarCells = ({ calendarDate, openModal }: CalendarCellsProps) => {
         const cellDate = getCellDateForWeekCalendar(calendarDate, i);
         const key = `${calendarDate.getDate()}-${calendarDate.getHours() + i}`;
 
+        const eventsForCell = getEventsForCell(cellDate, events);
```

Comment:
could you think of ways to improve this? 
Imagine that getEventsForCell takes 10 seconds : )